react-components/package.json

```diff
@@ -32,17 +32,22 @@
   "peerDependencies": {
     "@cognite/cogs-lab": "^9.0.0-alpha.153",
     "@cognite/reveal": "4.23.2",
+    "@cognite/sdk": "^9.13.0",
```

Comment:
Won't this cause trouble when linking to projects that have another SDK version? The newest SDK version is 10 (and it is used in Fusion, I believe)

Reply:
I mean currently we don't define the dependency at all... By declaring it as a peer dependency, fusion will at least be able to provide its version (which is the current behaviour).